src/main/java/org/spectrumauctions/sats/core/bidfile/CatsExporter.java

```diff
@@ -35,7 +36,7 @@ public File writeSingleBidderXOR(XORLanguage<? extends Good> valueFunction, int
             XORValue<?> value = (XORValue) iter.next();
             selectedValues.add(value);
             String line = String.valueOf(i).concat("\t");
-            line = line.concat(roundedValue(value.value().doubleValue()));
+            line = line.concat(value.value().setScale(4, BigDecimal.ROUND_HALF_UP).toString());
```

Comment:
We might think about replacing this with some rounding that takes the number of significant digits into account...

Reply:
No no, does not have to be very soon.
I was thinking that 0.000001 should not be rounded to 0. ;-) But the current models create bigger values anyway, hence this is not urgent.